admin/views/js-templates-primary-term.php

```diff
@@ -21,17 +21,17 @@
 
 <script type="text/html" id="tmpl-primary-term-ui">
 	<?php
-		printf(
-			'<button type="button" class="wpseo-make-primary-term" aria-label="%1$s">%2$s</button>',
-			esc_attr( sprintf(
-				/* translators: accessibility text. %1$s expands to the term title, %2$s to the taxonomy title. */
-				__( 'Make %1$s primary %2$s', 'wordpress-seo' ),
-				'{{data.term}}',
-				'{{data.taxonomy.title}}'
-			) ),
-			__( 'Make primary', 'wordpress-seo' )
-		);
-		?>
+	printf(
+		'<button type="button" class="wpseo-make-primary-term" aria-label="%1$s">%2$s</button>',
+		esc_attr( sprintf(
```

Comment:
Maybe it's possible to use `esc_attr__`

Reply:
Good question. I wasn't sure if the `{{..}}` would be ok without escaping, quite apart from it always being better to escape a complete string instead of parts of a string as there may be escape characters at the beginning/end of a string which could invalidate escaping if done in split up parts. That's why I'd left it as it was in this case.

Reply:
I see, lets leave this as it it for now 👍